examples/angular/src/app/app.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, ViewChild } from '@angular/core';
+import { init, locations } from '@contentful/app-sdk';
+import { DynamicComponentDirective } from './directives/dynamic-component.directive';
+import { DynamicComponent } from './dynamic-component';
+import { AppConfigLocation } from './AppConfigLocation/appConfigLocation.component';
+import { EntryFieldLocation } from './EntryFieldLocation/entryFieldLocation.component';
+import { Type } from '@angular/core';
+import { SidebarLocation } from './SidebarLocation/sidebarLocation.component';
+import { DialogLocation } from './DialogLocation/dialogLocation.component';
+import { EntryLocation } from './EntryLocation/entryLocation.component';
+import { PageLocation } from './PageLocation/pageLocation.component';
+import { HomeLocation } from './HomeLocation/homeLocation.component';
+import { LocalHostWarning } from './LocalHostWarning/localHostWarning.component';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  @ViewChild(DynamicComponentDirective, { static: true })
```

Comment:
Do you need a directive here? Feels overly complex. Shouldn't an element reference be fine?